Vagrantfile

```diff
@@ -98,18 +76,16 @@ SCRIPT
 
   # Use 1Gb of RAM for Vagrant box (otherwise bundle will go to swap)
   config.vm.provider :virtualbox do |vb|
-
-    vb.customize ['modifyvm', :id, '--memory', '2048']
+    vb.customize ['modifyvm', :id, '--memory', dev_mem]
+    vb.customize ['modifyvm', :id, '--cpus', dev_cpu]
```

Comment:
This will not make VirtualBox use more than once host CPU. This will just emulate more than one CPU :-)